src/StarterSite.php

```diff
@@ -35,6 +36,18 @@ public function __construct() {
 		parent::__construct();
 	}
 
+	/**
+	 * This enqueues theme styles.
+	 */
+	public function enqueue_styles() {
+		wp_enqueue_style(
+			'timber-starter-style',
+			get_template_directory_uri() . '/assets/styles/main.css',
+			[],
+			filemtime( get_template_directory() . '/assets/styles/main.css' )
```

Comment:
Could you put this into a variable? `get_template_directory_uri() . '/assets/styles/main.css'`